examples/sftp_get.py

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
This file and folder is not required. Please remove it.
We are planning to add a user guide later under docs that should examples with explanation.

Reply:
okay. Cool. I'll remove it.